app/models/LockStatus.scala

```diff
@@ -0,0 +1,10 @@
+package models
+
+import java.time.OffsetDateTime
+
+case class LockStatus(locked: Boolean, email: Option[String], timestamp: Option[OffsetDateTime])
```

Comment:
contents of this file is copied from `LockableS3ObjectController.scala` for sharing